frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -1166,6 +1166,13 @@ const plugin: Plugin<ConsumedExtensions> = [
       callback: doConnectsToBinding,
     },
   },
+  {
+    type: 'PostFormSubmissionAction',
+    properties: {
+      type: CONNECTOR_INCONTEXT_ACTIONS.serviceBinding,
+      callback: doContextualBinding,
+    },
+  },
```

Comment:
I'd rather this be in `operatorsTopologyPlugin` to keep it with other service binding extension support (eventually moved out of dev-console and into another mono repo).
Should this be dependent on the `ALLOW_SERVICE_BINDING` flag?

Reply:
This is just a callback. the actual action is dispatched from the context menu through URL params. Not sure if we should add the flag here.